meter_test.go

```diff
@@ -300,14 +321,11 @@ func TestMetricsGetJSONGlobPatterns(t *testing.T) {
 	require.Nil(t, metrics.Register("aaac", counter4))
 
 	for _, tCase := range tests {
-		b, err := metrics.GetJSON(tCase.pattern)
+		g := glob.MustCompile(tCase.pattern)
```

Comment:
Could you please separate the glob related tests from the general `meter` tests?
I'd suggest to put them under `_examples/glob_test.go`.